docs/development/i18n.rst

```diff
@@ -161,23 +161,11 @@ to the ``develop`` branch via pull requests for merge on a regular basis.
 
       $ git clone https://github.com/freedomofpress/securedrop
       $ cd securedrop
-      $ git remote add lab http://lab.securedrop.club/bot/securedrop/tree/i18n
-      $ git fetch lab
       $ git checkout -b wip-i18n origin/develop
```

Comment:
Before we sync from weblate, what about adding a note (maybe in a subsection "Adding a New Supported Language" or similar) which basically says:

1. Modify `SUPPORTED_LANGUAGES` in `securedrop/i18n_tool.py` to add the new language
2. Run `securedrop/bin/dev-shell ./i18n_tool.py update-docs`

Reply:
Ooops, missed this part. Added a paragraph in the documentation.